config/drupal-8/drupal-8-all-deprecations.php

```diff
@@ -11,7 +11,6 @@
 
     $containerConfigurator->import(PHPUnitSetList::PHPUNIT_60);
     $containerConfigurator->import(PHPUnitSetList::PHPUNIT_70);
-    $containerConfigurator->import(PHPUnitSetList::PHPUNIT_75);
```

Comment:
I should revert this and we fix it directly in https://www.drupal.org/project/rector/issues/3277704